File: src/main/java/videostore/horror/Movie.java
package videostore.horror;

public class Movie {
	private final String title;

	private final int priceCode;

	public Movie(String title, int priceCode) {
		this.title = title;
		this.priceCode = priceCode;
	}

	public int getPriceCode() {
		return priceCode;
	}


	public String getTitle() {
		return title;
	}
}

File: cs/1_Original/Soat.CleanCode.VideoStore.Original/Movie.cs
namespace Soat.CleanCode.VideoStore.Original
{
    public class Movie
    {
        public const int REGULAR     = 0;
        public const int NEW_RELEASE = 1;
        public const int CHILDREN    = 2;

        public int PriceCode { get; set; }
        public virtual string Title { get; }

        public Movie(string title, int priceCode)
        {
            Title     = title;
            PriceCode = priceCode;
            //comment
        }
    }
}
